geoportal/c2cgeoportal_geoportal/scaffolds/create/{{cookiecutter.project}}/env.default

```diff
@@ -39,7 +39,7 @@ TILECLOUDCHAIN_INTERNAL_PORT=8080
 # For internal print
 MAPSERVER_URL=http://mapserver:8080/
 TINYOWS_URL=http://tinyows:8080/
-QGISSERVER_URL=http://qgisserver:8080/
+QGISSERVER_URL=http://qgisserver:8080/mapserv_proxy
```

Comment:
Are you sure about this ? In all cases ?

Reply:
Yes, I'm pretty sure :-)